### PR TITLE
add 128 and 124 proficiency exams

### PR DESCRIPTION
**Pull request by rzig**

### CODE REVIEW

File: docs/Proficiency Exams/index.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # Overview
 
-Coming soon!
\ No newline at end of file
+For students coming in with prior experience or who are looking to graduate early, proficiency exams offer a way to skip classes. Proficiency exams are generally offered at the beginning of each semester and are graded on a pass-fail basis with no penalty for failing. Exams are typically three hours, and consist of a mixture of multiple-choice and free-response questions—they feel similar to AP exams. Proficiency exams are offered for many introductory courses, and students should contact their department if they have exceptional circumstances that would warrant a proficiency exam for an advanced course.
```
Comment (alex-m-zhang) [inline docs/Proficiency Exams/index.md]: There is some penalty for failing - iirc you can't retake proficiency exams too many times. Not sure if that's worth adding here though.

File: docs/Proficiency Exams/CS124.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# CS124
+
+[CS124 (Introduction to Computer Science I)](../Course%20Wiki/CS%20Course%20Offerings/CS124.md) is the first class in the CS core curriculum, and covers basic programming that many students are already familiar with.
+
+## Who Should Take It
```
Comment (alex-m-zhang) [inline docs/Proficiency Exams/CS124.md]: It -> the Proficiency Exam (just to make it clear)

File: docs/Proficiency Exams/CS128.md [hunk 3]
```diff
@@ -1,3 +1,40 @@
 # CS128
 
-Pain
\ No newline at end of file
+[CS128 (Introduction to Computer Science II)](../Course%20Wiki/CS%20Course%20Offerings/CS128.md) is the second programming course in the Computer Science core curriculum, and covers fundamentals of object-oriented C++ programming.
+
+## Who Should Take It
```
Comment (alex-m-zhang) [inline docs/Proficiency Exams/CS128.md]: It -> Proficiency Exam

**Review by alex-m-zhang — CHANGES_REQUESTED** [review]
A few changes, but nothing major. @lnguy7 is there anything else we want to add for proficiency exam sections? I think Ryan covered everything but I would like your opinion too.

**Review by lnguy7 — APPROVED** [review]
LGTM CS128 prof exam sounds like pain

**Review by alex-m-zhang — APPROVED** [review]
LGTM!

**Pull request merged**